.github/workflows/build_and_publish.yaml

```diff
@@ -73,6 +74,7 @@ jobs:
         run: |
           make version
           echo "value=$(cat dist/version)" >> "$GITHUB_OUTPUT"
+          echo "bare_value=$(cat dist/version | cut -d- -f1)" >> "$GITHUB_OUTPUT"
```

Comment:
I think this might be a problem if we ever do a pre-release, that is, a tag that itself contains `-`. A way to sidestep this would be to modify the `version` script, e.g. adding handling for a `--short` argument so it doesn't pass `--long` to `git` if `--short` is passed.

That being said, us creating tags with dashes sounds a bit unlikely so unless that bothers you, we can move forward.

Reply:
I went backwards and forwards on how to get the version without the extras. Eventually I settled on this after seeing it used here; https://github.com/grafana/synthetic-monitoring-agent/blob/main/scripts/release#L33

So I think we will have issues in other places if versions include a hyphen. However, that is probably a good reason to capture this logic (like you said) and put it in a in a script and reuse it.